addons/project/tests/test_project_recurrence.py

```diff
@@ -569,12 +570,23 @@ def get_task_and_subtask_counts(domain):
             'project_id': self.project_recurring.id
         })
         domain = [('project_id', '=', self.project_recurring.id)]
+        child_task_1 = self.env['project.task'].create({
+            'name': 'Child task 1',
+        })
+        child_task_2_recurrence = self.env['project.task'].create({
+            'name': 'Child task 2 that have recurrence',
+        })
         with Form(parent_task.with_context({'tracking_disable': True})) as task_form:
-            with task_form.child_ids.new() as subtask_form:
-                subtask_form.name = 'Child task 1'
-        with Form(parent_task.with_context({'tracking_disable': True})) as task_form:
-            with task_form.child_ids.new() as subtask_form:
-                subtask_form.name = 'Child task 2 that have recurrence'
+            task_form.child_ids.add(child_task_1)
+            task_form.child_ids.add(child_task_2_recurrence)
+            task_form.save()
```

Comment:
Normally the save method is automatically called at the end of the `with` statement.
```suggestion
```